src/features/messenger/conversations/CreateGroupDialog.tsx

```
@@ -0,0 +1,224 @@
+                          })}
+                        </DropdownMenuContent>
+                      </DropdownMenu>
+                    </FormControl>
+
+                    <FormMessage />
+                  </FormItem>
+                )
+              }}
+            />
+
+            {members && members.length ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)